extensions/ql-vscode/src/run-queries.ts

```diff
@@ -207,6 +207,9 @@ export class QueryEvaluationInfo {
             }
             void qs.logger.log(' --- Evaluator Log Summary --- ');
             void qs.logger.log(buffer.toString());
+            // Write summary to Query Log as well, as this information was present here before structured logging.
+            void logger.log(' --- Evaluator Log Summary --- ', { additionalLogLocation: this.logPath });
+            void logger.log(buffer.toString(), { additionalLogLocation: this.logPath });
```

Comment:
Note that this will also log to the "CodeQL Extension Log" console view as well, so this information is showing in many different places (Query Server console and Structured Evaluator Log (Summary), and now Query Log and Extension Log). Does this seem too noisy?

Reply:
Hmmm...not sure. It seems like query information is best kept in the query log. But, if people are asking for it in the extension log as well, that's probably fine.

Reply:
A ha, I see. I can just continue to use the query server `logger` object and log to an additional location of the query log (as requested).